ansible/environments/docker-machine/hosts.j2.ini

```diff
@@ -8,17 +8,24 @@ ansible ansible_connection=local
 
 [controllers]
 controller0                 ansible_host={{ docker_machine_ip }}
+{% if mode is defined and 'HA' in mode %}
```

Comment:
Running setup with `-e mode=HA` will replicate the components.
Using wskdev: `wskdev setup -a "-e mode=HA"` will do that also. Off by default for local dev.